Aplib.Core/Belief/Belief.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+
+namespace Aplib.Core.Belief
+{
+    /// <summary>
+    /// The <see cref="Belief{TReference, TResource}"/> class represents a belief.
+    /// Some <i>object reference</i> is used to generate/update a <i>resource</i> 
+    /// (i.e., some piece of information on the game state as perceived by an agent).
+    /// It implements the <see cref="IBelief"/> interface.
+    /// It supports implicit conversion to <typeparamref name="TResource"/>.
+    /// </summary>
```

Comment:
These lines are perfect for a `remark`
```suggestion
    /// </summary>
    /// <remark>
    /// It implements the <see cref="IBelief"/> interface.
    /// It supports implicit conversion to <typeparamref name="TResource"/>.
    /// </remark>
```

Reply:
Good suggestion!